cmd/advertisement-operator/main.go

```diff
@@ -80,12 +80,17 @@ func main() {
 	var liqoNamespace, kubeletImage, initKubeletImage string
 	var resyncPeriod int64
 	var offloadingStatusControllerRequeueTime int64
+	var offerUploadThreshold int64
+	var timeToLive int64
 
 	flag.StringVar(&metricsAddr, "metrics-addr", defaultMetricsaddr, "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection,
 		"enable-leader-election", false,
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
+	flag.Int64Var(&offerUploadThreshold, "offer-upload-threshold-perc", int64(5),
```

Comment:
```suggestion
	flag.Int64Var(&offerUpdateThreshold, "offer-update-threshold-perc", int64(5),
```

Should be probably unsigned, and would add a sanity check below to ensure it does not exceed 100.